importers/src/BeImportFileCreator.class.ts

```diff
@@ -129,7 +129,7 @@ export class BeImportFileCreator {
 
             // pack everything
             const zipArchive = archiver('zip');
-            zipArchive.on('warning', function(err) {
+            zipArchive.on('warning', function (err) {
```

Comment:
nit: we should have a linter and have CI enforce this, so these formatting things are dealt with in one PR and then we have vscode settings that respect this

Reply:
That makes sense. Are you also using prettier? Would you like to set up a vscode-settings & linter as suggested?

One issue I'm seeing that prettier seems to change its formatting from time to time.  Not sure if that is a problem on my side